includes/class-wsu-syndicate-shortcode-json.php

```diff
@@ -293,39 +303,46 @@ public function process_remote_posts( $data, $atts ) {
 
 		foreach ( $data as $post ) {
 			$subset = new StdClass();
-			$subset->ID = $post->id;
-			$subset->date = $post->date; // In time zone of requested site
-			$subset->link = $post->link;
-
-			// These fields all provide a rendered version when the response is generated.
-			$subset->title   = $post->title->rendered;
-			$subset->content = $post->content->rendered;
-			$subset->excerpt = $post->excerpt->rendered;
-
-			// If a featured image is assigned (int), the full data will be in the `_embedded` property.
-			if ( ! empty( $post->featured_media ) && isset( $post->_embedded->{'wp:featuredmedia'} ) && 0 < count( $post->_embedded->{'wp:featuredmedia'} ) ) {
-				$subset->featured_media = $post->_embedded->{'wp:featuredmedia'}[0];
-
-				if ( isset( $subset->featured_media->media_details->sizes->{'post-thumbnail'} ) ) {
-					$subset->thumbnail = $subset->featured_media->media_details->sizes->{'post-thumbnail'}->source_url;
-				} elseif ( isset( $subset->featured_media->media_details->sizes->{'thumbnail'} ) ) {
-					$subset->thumbnail = $subset->featured_media->media_details->sizes->{'thumbnail'}->source_url;
+
+			// Only a subset of data is returned for a headlines request.
+			if ( 'headlines' === $atts['output'] ) {
+				$subset->link = $post->link;
+				$subset->title   = $post->title->rendered;
```

Comment:
There are a couple extra spaces here between `title` and `=`.

Same with line 399 (and 317 and 406, but those may be intentional).